### DISCUSSION TITLE
Cannot start LANraragi after updating [FreeBSD]

### DISCUSSION THREAD

**Discussion by bun-dev** [Difegue/LANraragi#867] [Q&A] [answered]
I'm not sure what I did wrong. I did a pkg update/upgrade and git pull, and now I can't start LANraragi at all. After running npm start, it throws an error. This is running in a jail for FreeBSD(TrueNas). It was previously working fine.

`> lanraragi@0.8.90 start
> perl ./script/launcher.pl -f ./script/lanraragi

Server PID will be at /media/tsu/LANraragi/public/temp/server.pid
Can't load application from file "/media/tsu/LANraragi/script/lanraragi": /media/tsu/LANraragi/script/lanraragi did not return a true value at /usr/local/lib/perl5/site_perl/Mojo/Server.pm line 59.
`

I even tried with a clean installation. Same problem.

**Comment by Difegue** [comment]
No idea what's going wrong here I'm afraid -- The matching line in mojo's server code is here: https://github.com/mojolicious/mojo/blob/a607e42d80ba169299375e911577fdf4abe28540/lib/Mojo/Server.pm#L59 but it's not very helpful.  

Have you tried running it jail-less just to check? There might be some new stuff that has to be done w/ the jail to get things working since you updated.

**Reply by bun-dev** [reply, marked as answer]
It seems the issue was related to a borked update process in my jail.  I'm not sure why this happened, which makes me a bit worried about doing a bsd jail update in the future.  In any case, I managed to get it working again by creating a new jail from scratch. Thanks for the response!
